src/Datadog.Trace.ClrProfiler.Managed/Integrations/HttpMessageHandlerIntegration.cs

```diff
@@ -0,0 +1,113 @@
+using System;
```

Comment:
Seems this was pulled from the HTTP PR, which covers this and others in here from that PR